docker-compose-postgres.yaml

```diff
@@ -1,12 +1,12 @@
-version: '3'
+version: '3.8'
```

Comment:
3.8 version https://docs.docker.com/compose/compose-file/compose-file-v3/#variable-substitution did't work for me. Looks like it is applicable to only variable block where the name of a key is same across the services, which is not in our case. Therefore falling back to the primitive use of .env file, where we substitute just the value of whatever key the service is expecting